doc/radosgw/s3select.rst

```diff
@@ -261,25 +262,24 @@ s3-select function interfaces
 Timestamp functions
 ~~~~~~~~~~~~~~~~~~~
     | The timestamp functionalities as described in `AWS-specs <https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-date.html>`_  is fully implemented.
-    | the casting operator( ``to_timestamp( string )`` ), converts string to timestamp basic type.
-    | to_timestamp operator able to convert the following ``YYYY-MM-DD-HH-MI-SS.sssss(micro secs)+-HH:MI`` string format into timestamp
-    | upon time(or part of it) is missing, zero's are replacing the missing parts.
-    | +/- HH:MI is also optional, in the case it's missing, it's considere as 00:00. 
-    | values between -12:00 and 14:00 set the timezone of the timestamp value. 
 
-    | ``extract(date-part, timestamp)`` : function return integer according to date-part extract from input timestamp.
-    | supported date-part : year,month,week,day,hour,minute,second,timezone_hour,timezone_minute.
+    | ``to_timestamp( string )`` : The casting operator converts string to timestamp basic type.
+    | to_timestamp operator is able to convert the following ``YYYY-MM-DDTHH:mm:ss.SSSSSS+/-HH:mm`` , ``YYYY-MM-DDTHH:mm:ss.SSSSSSZ`` , ``YYYY-MM-DDTHH:mm:ss+/-HH:mm`` , ``YYYY-MM-DDTHH:mm:ssZ`` , ``YYYY-MM-DDTHH:mm+/-HH:mm`` , ``YYYY-MM-DDTHH:mmZ`` , ``YYYY-MM-DDT`` or ``YYYYT`` string formats into timestamp.
+    | Where time(or part of it) is missing in the string format, zero's are replacing the missing parts. And for missing month and day, 1 is default value for them.
+    | Timezone part is in format ``+/-HH:mm`` or ``Z`` , where the letter "Z" indicates Coordinated Universal Time (UTC). Value of timezone can range between -12:00 and +14:00.
```

Comment:
:+1:

Reply:
space needed between time and (or part of it)?

Reply:
in timestamp, the format is quite stiff (no space is allowed, it will cause a failure)

Reply:
this update move to #41834